Cargo.toml

```diff
@@ -40,10 +40,13 @@ semver = { workspace = true, features = ["serde"] }
 tokio = { workspace = true, features = ["full"] }
 
 [features]
-default = ["full"]
+default = ["full", "tls-openssl"]
 full = ["core", "extra"]
 core = ["logger", "process-monitor", "network-monitor", "file-system-monitor"]
 extra = ["rules-engine", "desktop-notifier", "smtp-notifier"]
+static = ["all-vendored", "tls-rustls"]
```

Comment:
I prefer the tls implementation being explicit and not hidden under this new feature.